controller/test.js

```diff
@@ -169,6 +169,13 @@ function displayResults({ testResults }) {
       console.log(`> > > ${stackLineTwo}`.grey);
     });
 
+    //part divided by the whole 3/4 = 75%
+    //use math .round to round to the nearest tenth of a percent
+
+    console.log(
+      clc.green(`You have passed ${ 100 * (testResults.stats.passes / testResults.stats.tests) }% of the test.`)
```

Comment:
Is this `clc.green` line running regardless of the percentage of tests passed? Do we want a green message displaying that 20% of tests are passing, or should that be a  red message? How can we conditionally choose which `clc` method to use so that we get an appropriate colored message depending on the tests passing?
```
100% of tests passing --> `clc.green`
75% or more tests passing --> `clc.yellow`
less than 75% of tests passing --> `clc.red`
```

Reply:
Yes sir, got it.